xc/common/utils/animate_router_pop.py

```diff
@@ -30,13 +30,13 @@ def main():
 
     final_route = []
 
-    for l in sys.stdin:
```

Comment:
Please restrict all changes to the `quicklogic` folder. Files under `xc` are related only to Xilinx FPGA support and are not used for QuickLogic FPGAs in any way. You may disable linting them in the Travis CI.

If you change these files it would make it harder to integrate QuickLogic support with upstream SymbiFlow in the future.